app/code/Magento/Backend/Controller/Adminhtml/Auth/Login.php

```diff
@@ -53,6 +62,7 @@ public function execute()
         } else {
             return $this->resultPageFactory->create();
         }
+        */
```

Comment:
Looks like such change will cause regression of https://github.com/magento/magento2/commit/2bf7c958f68137ee187f8798c73aed15846a1a33#diff-7588992600e1c858efa3a6a4c7b31cc2

Please check and rework a fix so that both cases work fine.